test/cases/active_job_serializer_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'abstract_unit'
+
+require 'fixtures/project'
+require 'fixtures/person'
+require 'active_job_object_serializer'
+require 'active_resource/active_job_serializer'
+
+class ActiveJobSerializerTest < ActiveSupport::TestCase
+
+  def setup
+    @klass = ActiveResource::ActiveJobSerializer
+    @person = Person.new({ id: 2, name: 'David' }).tap do |person|
+      person.persisted = true
+    end
+    @person_json = jsonify(@person)
+    @project = Project.new({ id: 1, name: 'Ruby on Rails' }).tap do |project|
```

Comment:
Do we need the braces around here and above?

I'd like to some line breaks as well. How about this?

```ruby
setup do
  @person = Person.new(id: 2, name: 'David')
  @person.persisted = true

  @project = Project.new(id: 1, name: 'Ruby on Rails')
  @project.prefix_options[:person_id] = @person.id
end
```